BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -325,8 +325,7 @@ - (NSString *)fullFolderFieldsParameterString
                        BOXAPIObjectKeyExtension,
                        BOXAPIObjectKeyIsPackage,
                        BOXAPIObjectKeyAllowedSharedLinkAccessLevels,
-                       BOXAPIObjectKeyCollections,
-                       BOXAPIObjectKeyRepresentations];
+                       BOXAPIObjectKeyCollections];
```

Comment:
what's the impact of this change? I thought we'd already made the performance related change on this, is this in addition or was my understanding incorrect?